### PR TITLE
Unify remote-repositories parameter

### PR DESCRIPTION
**Pull request by spyrkob**
`--repositories` overrides any repositories already configured either in installed server, or known-feature-packs.
In operations on installed server (update, revert) the repositories defined that way are not persisted. 
The values permitted are either comma-separated URLs or pairs or ID::URL

Renamed `--local-repo` to `--local-cache` and `--channel` to `--manifest`

### CODE REVIEW

**Comment by spyrkob** [comment]
@TomasHofman could you take a look, do the CLI changes make sense to you?

File: prospero-common/src/main/java/org/wildfly/prospero/api/TemporaryRepositoriesHandler.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            final List<Repository> repositories = mergeRepositories(oc.getRepositories(), additionalRepositories);
+            final Channel c = new Channel(oc.getSchemaVersion(), oc.getName(), oc.getDescription(), oc.getVendor(),
+                    oc.getChannelRequirements(), repositories, oc.getManifestRef());
+            mergedChannels.add(c);
+        }
+
+        return mergedChannels;
+    }
+
+    private static List<Repository> mergeRepositories(List<Repository> originalRepositories, List<Repository> additionalRepositories) {
```
Comment (TomasHofman) [inline prospero-common/src/main/java/org/wildfly/prospero/api/TemporaryRepositoriesHandler.java:49]: Should it be called `mergeRepositories` when it overrides them? Any plans for this behavior to be modifiable?

**Comment by TomasHofman** [comment]
Seems OK to me after a quick read. I think it's a good change.

File: prospero-common/src/main/java/org/wildfly/prospero/api/TemporaryRepositoriesHandler.java (hunk 1 above)
Comment (spyrkob) [reply prospero-common/src/main/java/org/wildfly/prospero/api/TemporaryRepositoriesHandler.java:49]: Good catch, that's leftover from some experiments. I'll remove it

**Pull request merged**